frontend/libs/studio-components/src/components/StudioCheckboxTable/StudioCheckboxTableContext/StudioCheckboxTableContext.test.tsx

```
@@ -0,0 +1,47 @@
+    render(
+      <StudioCheckboxTableContextProvider>
+        <button>My button</button>
+      </StudioCheckboxTableContextProvider>,
+    );
+
+    expect(screen.getByRole('button', { name: 'My button' })).toBeInTheDocument();
+  });
+
+  it('should provide a useCheckboxTableContext hook', () => {
+    const TestComponent = (): ReactElement => {
+      const {} = useCheckboxTableContext();
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)